openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfile.java

```diff
@@ -625,23 +627,33 @@ public MatrixResult computeMatrix(MatrixRequest req) throws Exception {
 
         try {
             HintsMap hintsMap = new HintsMap();
-            int weightingMethod = req.getWeightingMethod() == WeightingMethod.UNKNOWN ? WeightingMethod.RECOMMENDED : req.getWeightingMethod();
+            EdgeFilter edgeFilter = DefaultEdgeFilter.allEdges(flagEncoder);
+            int weightingMethod = req.getWeightingMethod() == WeightingMethod.UNKNOWN ? WeightingMethod.FASTEST : req.getWeightingMethod();
```

Comment:
Why did the default changed from `WeightingMethod.RECOMMENDED` to `WeightingMethod.FASTEST`?

Reply:
Ups. done

Reply:
No worries :)